pkg/openthread/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_OPENTHREAD
+    bool "Openthread network stack"
+    depends on TEST_KCONFIG
+
+    depends on HAS_CPP
+    select MODULE_CPP11-COMPAT
+    select MODULE_EVENT
+    select MODULE_NETDEV
+    select MODULE_OPENTHREAD_CONTRIB
+    select MODULE_OPENTHREAD_CONTRIB_NETDEV
+    select MODULE_L2UTIL
+    select MODULE_TIMEX
+    select MODULE_ZTIMER
+    select MODULE_ZTIMER64
+    select MODULE_ZTIMER_MSEC
+
+config MODULE_OPENTHREAD_CONTRIB
+    bool "Openthread contrib layer"
+
+config MODULE_OPENTHREAD_CONTRIB_NETDEV
+    bool "Openthread netdev contrib layer"
```

Comment:
These modules are never optional, right? I think we can remove the prompt here, and add a dependency to `PACKAGE_OPENTHREAD`.